flecs/edge_set.py

```diff
@@ -54,7 +55,12 @@ def __getattr__(self, item):
         if item in self.attribute_dict.keys():
             return self.attribute_dict[item]
         else:
-            raise AttributeError
+            return super().__getattr__(item)
+
+    def __setattr__(self, key, value):
```

Comment:
The inheritance from `torch.nn.Module` introduced some issues with the `__setattr__` method, which are now fixed.